server/routes.js

```diff
@@ -27,6 +30,10 @@ function configureRoutes(app) {
   app.post('/api/sirenAddressed', siren.validateSirenAddressed, siren.handleSirenAddressed)
   app.post('/api/sirenEscalated', siren.validateSirenEscalated, siren.handleSirenEscalated)
 
+  app.post('/pa/create-sensor-location', clickUpHelpers.clickUpChecker, pa.handleCreateSensorLocation)
```

Comment:
I haven't seen this way of using express before, and can't seem to quite put it into words to find documentation on it. Out of curiosity, does it just call the functions successively until a response is sent? I like how it's infinitely cleaner.

Reply:
Here's some good documentation: https://expressjs.com/en/guide/using-middleware.html